src/calendar/day/index.tsx

```diff
@@ -78,15 +77,15 @@ export default class Day extends Component<DayProps> {
   }
 
   getAccessibilityLabel = memoize((day, marking, isToday) => {
-    const today = get(XDate, 'locales[XDate.defaultLocale].today');
-    const formatAccessibilityLabel = get(XDate, 'locales[XDate.defaultLocale].formatAccessibilityLabel');
+    const today = XDate.locales[XDate.defaultLocale].today || 'today';
+    const formatAccessibilityLabel = XDate.locales[XDate.defaultLocale].formatAccessibilityLabel;
```

Comment:
You'll need to resolve ts errors as both 'today' and 'formatAccessibilityLabel'  does not exist on type 'locale_detail'

Reply:
Fixed!